wagtail/admin/templates/wagtailadmin/shared/keyboard_shortcuts_dialog.html

```diff
@@ -0,0 +1,30 @@
+{% load wagtailadmin_tags i18n %}
+{% dialog icon_name='regex' id='keyboard-shortcuts-dialog' title=_("Keyboard shortcuts") %}
+    {% if shortcuts%}
+        <table class="w-keyboard-shortcuts">
+            <caption class="w-sr-only">
+                {% trans "All keyboard shortcuts" %}
+            </caption>
+            <thead class="w-sr-only">
+                <tr>
+                    <th scope="col">{% trans "Section" %}</th>
+                    <th scope="col">{% trans "Keyboard shortcut" %}</th>
+                </tr>
+            </thead>
+
+            {% for category, shortcuts in shortcuts.items %}
+                <tbody class="w-w-full" id="keyboard-shortut-group-{{ category.0 }}">
+                    <tr colspan="2">
+                        <th class="w-keyboard-shortcuts__category" scope="rowgroup">{{ category.1 }}</th>
+                    </tr>
+                    {% for shortcut in shortcuts %}
+                        <tr class="w-keyboard_shortcuts__key ">
```

Comment:
```suggestion
                        <tr class="w-keyboard_shortcuts__key">
```

Just a small class whitespace cleanup